LLMonFHIR.xcodeproj/project.pbxproj

```diff
@@ -798,7 +802,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = "LLMonFHIR/Supporting Files/LLMonFHIR.entitlements";
 				CODE_SIGN_IDENTITY = "Apple Development";
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_STYLE = Manual;
```

Comment:
Please ensure that the release settings are the same as before, probably the easiest way to do this is to change this line in a text editor.